src/Policies.h

```diff
@@ -31,17 +31,15 @@ class Policy : public AbstractRunnableModule
 {
     static constexpr size_t HANDLERS_SEQUENCE_SIZE = 1;
 
-    Tap *_tap;
+    std::vector<Tap *> _tap;
```

Comment:
i _think_ precedent is that this and _input_stream should be plural now, so `_taps`? let's see what we do elsewhere if you haven't

Reply:
done